spec/fixtures/user_mailer/account_activation

```diff
@@ -0,0 +1,3 @@
+UserMailer#account_activation
```

Comment:
I think you can delete this or change the message

Reply:
Done!